src/components/splitButton/splitButton.scss

```diff
@@ -0,0 +1,308 @@
+    padding: 16px 16px;
+    height: 56px;
+    &-arrow-btn {
+      min-width: 56px;
+      width: 56px;
+    }
+  }
+}
+
+// Disabled Button Styles
+@mixin kd-split-btn-disabled {
+  background-color: var(--kd-color-background-disabled);
+  color: var(--kd-color-text-inversed);
+  border: none;
+  cursor: not-allowed;
+  pointer-events: none;
+}
+
+@mixin kd-split-btn-focus {
+  outline-color: var(--kd-color-border-focus);
+}
+
+span {
+  display: flex;
+  align-items: center;
+  z-index: 1;
+}
+
+// Primary Split Button Styles (Web & App)
+.kd-split-btn--primary-web,
+.kd-split-btn--primary-app {
+  @extend .kd-split-btn;
+  background-color: var(--kd-color-background-primary);
+  color: var(--kd-color-text-inversed);
+  border: none;
+  position: relative;
+  overflow: hidden;
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-color: var(--kd-color-background-primary-hover);
+    transition: transform 300ms ease-out, opacity 200ms ease-out,
+      background-color 150ms ease-in-out;
+    transform: scaleX(0);
+    transform-origin: left;
+    opacity: 0;
+  }
+
+  &:hover {
+    &:before {
+      transform: scaleX(1);
+      opacity: 1;
+    }
+  }
+
+  &:active {
+    border: none;
+
+    &:before {
+      background-color: var(--kd-color-background-primary-pressed);
+    }
+  }
+
+  &:focus {
+    @include kd-split-btn-focus;
+  }
+
+  &:disabled {
+    @include kd-split-btn-disabled;
+  }
+}
+
+.kd-split-btn--primary-web {
+  background-color: var(--kd-color-background-secondary);
+
+  &:before {
+    background-color: var(--kd-color-background-secondary-hover);
+  }
+
+  &:active:before {
+    background-color: var(--kd-color-background-secondary-pressed);
+  }
+}
+
+// Primary Destructive Button
+.kd-split-btn--primary-app-destructive,
+.kd-split-btn--primary-web-destructive {
+  @extend .kd-split-btn;
+  background-color: var(--kd-color-background-destructive);
+  color: var(--kd-color-text-inversed);
+  border: none;
+  position: relative;
+  overflow: hidden;
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-color: var(--kd-color-background-destructive-hover);
+    transition: transform 300ms ease-out, opacity 200ms ease-out,
+      background-color 150ms ease-in-out;
+    transform: scaleX(0);
+    transform-origin: left;
+    opacity: 0;
+  }
+
+  &:hover {
+    &:before {
+      transform: scaleX(1);
+      opacity: 1;
+    }
+  }
+
+  &:active:before {
+    background-color: var(--kd-color-background-destructive-pressed);
+  }
+
+  &:focus {
+    @include kd-split-btn-focus;
+  }
+
+  &:disabled {
+    @include kd-split-btn-disabled;
+  }
+}
+
+// Secondary Button Styles
+.kd-split-btn--secondary {
+  @extend .kd-split-btn;
+  background-color: var(--kd-color-transparent);
+  color: var(--kd-color-text-primary);
+  border: 1px solid var(--kd-color-border-primary);
```

Comment:
The secondary style has a 2px thick border between the buttons, needs to be 1px based on the design.

Reply:
Done